src/ontology/mondo-edit.obo

```diff
@@ -177066,6 +177068,7 @@ synonym: "encephalotrigeminal syndrome" EXACT [NCIT:C3391]
 synonym: "fourth phacomatosis" RELATED [GARD:0007706]
 synonym: "leptomeningeal angiomatosis" RELATED [GARD:0007706]
 synonym: "meningeal capillary angiomatosis" RELATED [GARD:0007706]
+synonym: "Sturge Weber Syndrome" EXACT [NORD:1741]
```

Comment:
This is a new synonym though there is another one (the one below) that is the same, but without the capitalization of "syndrome". I think we should address this capitalization issue at a later time. Making a comment just to bring our attention to this.

Reply:
Thanks for the note. I added a [ticket](https://github.com/monarch-initiative/mondo/issues/8171) about this. 

@sabrinatoro  Can I fix this now? Also, is there a way to contact NORD about the preferred letter case for the synonym, e.g. Sturge Weber syndrome?

Reply:
I removed the duplicate.